testsuite/billing.py

```diff
@@ -0,0 +1,53 @@
+"""APIs for Stripe and Braintree"""
+import braintree
+import stripe
+
+
+# pylint: disable=too-few-public-methods
+class Stripe:
+    """API for Stripe"""
+
+    def __init__(self, api_key):
+        stripe.api_key = api_key
+
+    @staticmethod
+    def read_payment(customer_id):
+        """Read Stripe payment"""
+        return list(filter(lambda x: x['customer'] == customer_id, stripe.PaymentIntent.list()))[0]
+
+    @staticmethod
+    def read_customer_by_account(account):
+        """Read Stripe customer"""
+        return list(filter(lambda x: str(account.entity_id) in x['metadata'].get("3scale_account_reference", []),
```

Comment:
list-filter-lambda combo is banned

`{x for x in stripe.Customer.list() if str(account.entity_id) in x["metadata"].get("3scale_account_reference", [])}[0]`